### ISSUE TITLE
Indent is not reset after first line break in jsdoc parser

### ISSUE THREAD

**Issue by andreasrueedlinger** [Microsoft/TypeScript#10987]
**TypeScript Version:**  nightly (2.1.0-dev.20160910)

**Code**
Adding a JSDoc comment to a method with linebreaks like this:

``` ts
/**
  * This is a multiline
  * comment and should be parsed correctly
  */
someMethod() {
}
```

But instead the JSDoc-Parser parses `This is a mutiline\ncommentand should be parsed correctly`

**Expected behavior:**
Parsed JSDoc: `This is a mutiline\ncomment and should be parsed correctly`
**Actual behavior:** 
Parsed JSDoc: `This is a mutiline\ncommentand should be parsed correctly`
Not the missing whitespace on the second line!

The problem is a missing reset of the indent variable after the first linebreak here:
https://github.com/Microsoft/TypeScript/blob/955f2f2fc0c77e458294520d102d4a0973506923/src/compiler/parser.ts#L6198

There should be `indent = 0;` as on Line 6222.

**Comment by mhegazy** [comment]
@andreasrueedlinger since you debugged through it, a PR would be appreciated.

**Comment by andreasrueedlinger** [comment]
@mhegazy sorry but a PR would be too much of an effort for me (not enough into it), but here is the patch:
```
diff --git a/src/compiler/parser.ts b/src/compiler/parser.ts
index 1aec63b..6bf89a1 100644
--- a/src/compiler/parser.ts
+++ b/src/compiler/parser.ts
@@ -6196,6 +6196,7 @@ namespace ts {
                     }
                     if (token() === SyntaxKind.NewLineTrivia) {
                         state = JSDocState.BeginningOfLine;
+                        indent = 0;
                         nextJSDocToken();
                     }
                     while (token() !== SyntaxKind.EndOfFileToken) {
```

**Comment by andreasrueedlinger** [comment]
@mhegazy is no one able to create a PR with this patch? Should not be too much work if you are into the typescript setup and the test framework?

**Comment by sandersn** [comment]
Sorry, I was heads-down on object spread and rest, so I missed this bug the first time around.
Unfortunately, I can't repro this bug on master given this code:

```ts
/**
  * This is a multiline
  * comment and should be parsed correctly
  */
function foo(x) {
}

foo()
```

Or this code:

```ts
class K {
  /**
   * This is a multiline
   * comment and should be parsed correctly
   */
  foo(x) {
  }
}

(new K()).foo()
```

I'll try again on release-2.0

**Comment by sandersn** [comment]
The parser code hasn't changed much since Sep 19 when you opened the bug. @andreasrueedlinger I guess I must be missing something important about the repro code?

**Comment by andreasrueedlinger** [comment]
Here is code that should reproduce the bug:
`package.json`:
```
{
  "name": "tsbug",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "scripts": {
    "start": "tsc index.ts && node index.js",
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "author": "",
  "license": "ISC",
  "dependencies": {
    "typescript": "2.1.1"
  }
}
```
`index.ts`:
```ts
import * as ts from 'typescript';

const program = ts.createProgram(['model.ts'], {});
const typeChecker = program.getTypeChecker();
const nodes: ts.Node[] = [];
program.getSourceFiles().forEach(sf => {
  ts.forEachChild(sf, node => {
    nodes.push(node);
  });
});

for (let node of nodes.filter(node => node.kind === ts.SyntaxKind.ClassDeclaration) as ts.ClassDeclaration[]) {
  node.members.filter(m => m.kind === ts.SyntaxKind.MethodDeclaration).forEach((m: ts.MethodDeclaration) => {
    const symbol = typeChecker.getSymbolAtLocation(m.name);
    const comments = symbol.getDocumentationComment();
    if (comments.length) { console.log(ts.displayPartsToString(comments)); }
  });
}
```
`model.ts`:
```ts
export class Test {
  /**
   * This is a description of the getModel method
   * this is some more text on another line
   */
  public someMethod() {
    return true;
  }
}
```
You can also compare to the previous working version: `npm i typescript@2.1.0-dev.20160910`

```

**Comment by sandersn** [comment]
Thanks. It only repros when the first space on the second line happens less than *n* spaces from the beginning of the indent, where *n* is the number of spaces between the beginning of the line and the beginning of the indent. 

My attempts at a repro start the second line with 'comment', putting the space 7 characters from the indent, and the indent was only 4 or 6.

**Comment by sandersn** [comment]
The fix is available for review at #12156 now
